Taskfile.yml

```diff
@@ -109,21 +109,21 @@ tasks:
     desc: "Run all integration tests in parallel"
     deps:
       - task: test-integration-db
-        vars: {DB: "FerretDB"}
+        vars: { DB: "FerretDB" }
```

Comment:
My yaml formatter is insisting on this change :) Should I revert it?

Reply:
`"yaml.format.bracketSpacing": false` in VS Code :)